### ISSUE TITLE
APO missing object type

### ISSUE THREAD

**Issue by justinlittman** [sul-dlss/argo#3152]
The create APO integration test is expecting the Details table to show an object type:  https://github.com/sul-dlss/infrastructure-integration-test/blob/main/spec/features/create_apo_spec.rb#L32-L33

However, the table no longer contains this:
![image](https://user-images.githubusercontent.com/588335/155527000-a439843a-c386-4fc7-8edc-9e4d55940cc0.png)

Unlike a collection:
![image](https://user-images.githubusercontent.com/588335/155527081-679175e5-850d-4980-aa65-ddfdf4b1acd8.png)

If this change in the test is intentional, the test needs to be changed; otherwise, the table needs to be fixed.

**Comment by jcoyne** [comment]
Yes, this is intentional per the mockups: https://projects.invisionapp.com/share/ZR1072MLCMEW#/screens/446060026

**Comment by andrewjbtw** [comment]
The object type is now in the pill.
